webapp-django/crashstats/base/jinja2/500.html

```diff
@@ -1,24 +1,19 @@
-{% extends "crashstats_base.html" %}
-
-{% block page_title %}
-Internal Server Error
-{% endblock %}
-
-{% block product_nav_filter %}&nbsp;{% endblock %}
-
+{% extends "error.html" %}
+{% block page_title %}Internal Server Error{% endblock %}</title>
 {% block content %}
 <div id="mainbody">
+    <div class="page-heading">
+        <h2>Internal Server Error</h2>
+    </div>
     <div class="panel">
         <div class="title">
-            <h2>Internal Server Error</h2>
+            <h2>Something went wrong, it is not you it's me.</h2>
         </div>
 
         <div class="body">
-            <p>Something went wrong, it is not you it's me</p>
-            <p>Please file an <a href="https://bugzilla.mozilla.org/enter_bug.cgi?{{ make_query_string(
+            <p>Please file an <a class="bugzilla-link" href="https://bugzilla.mozilla.org/enter_bug.cgi?{{ make_query_string(
                 product='Socorro',
                 component='Webapp',
-                bug_file_loc=request.build_absolute_uri(),
```

Comment:
You can't use `request` in this template. Django goes into a strange mode when you've ended up here. If you try to use anything from `request` (which I think comes from the context processor) you get an error (within the error). And that's what I'm trying to avoid.